### PR TITLE
{bio}[foss/2023a] infercnvpy v0.4.3

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/i/infercnvpy/infercnvpy-0.4.3-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+name = 'infercnvpy'
+version = '0.4.3'
+
+homepage = 'https://github.com/icbi-lab/infercnvpy'
+description = "Infer copy number variation (CNV) from scRNA-seq data. Plays nicely with Scanpy."
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [('hatchling', '1.18.0'),]
```
Comment (Micket) [inline easybuild/easyconfigs/i/infercnvpy/infercnvpy-0.4.3-foss-2023a.eb:11]:
```suggestion
builddependencies = [('hatchling', '1.18.0')]
```

**Review by Micket — CHANGES_REQUESTED** [review]

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 17 out of 17 (1 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/679efa4dea3222fd14e9e975d35b266e for a full test report.

**Comment by Micket** [comment]
Going in, thanks @pavelToman!

**Pull request merged**